addons/stock/models/stock_move.py

```diff
@@ -176,7 +176,10 @@ def _default_group_id(self):
     next_serial = fields.Char('First SN')
     next_serial_count = fields.Integer('Number of SN')
     orderpoint_id = fields.Many2one('stock.warehouse.orderpoint', 'Original Reordering Rule', check_company=True)
+    # TODO : remove `json_forecast` in master, use forcast_ fields instead
     json_forecast = fields.Char('JSON data for the forecast widget', compute='_compute_json_forecast')
+    forecast_availability = fields.Float('Forcast Availability', compute='_compute_json_forecast', digits='Product Unit of Measure')
```

Comment:
I'm wondering if this is not better to compute this field in the same compute than the reserved_availability one